scripts/install_deps.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 set -e
 
-gem install xcpretty
```

Comment:
should we install xcbeautify here? I think that is why the build failed...

Reply:
Odd. I had it in `Brewfile`, and agent complained it was already installed. 🤷‍♂ 

![Screen Shot 2020-04-06 at 11 17 11 pm](https://user-images.githubusercontent.com/1733327/78562482-c7a49d80-785c-11ea-8389-134992bd1c1f.png)

I'll add it in anyway. Better for it to complain to re-install instead of failing the build.